src/ray/core_worker/gcs_server_address_updater.h

```diff
@@ -37,13 +37,13 @@ class GcsServerAddressUpdater {
   /// Update gcs server address.
   void UpdateGcsServerAddress();
 
-  std::unique_ptr<rpc::ClientCallManager> client_call_manager_;
```

Comment:
This change just removes unnecessary pointer usage for cleaner destruction ordering.